pyseir/cli.py

```diff
@@ -187,14 +152,8 @@ def _build_all_for_states(
     web_ui_mapper = WebUIDataAdaptorV1(
         output_interval_days=output_interval_days, run_mode=run_mode, output_dir=output_dir,
     )
-    for state in states:
-        region = pipeline.Region.from_state(state)
-        state_input = webui_data_adaptor_v1.RegionalInput.from_region(region)
-        web_ui_mapper.generate_state(
-            state_input,
-            whitelisted_county_fips=[k for k, v in all_county_fips.items() if v == state],
-            states_only=False,
-        )
+    for fitter in itertools.chain(state_fitters, county_fitters):
+        web_ui_mapper.map_fips(webui_data_adaptor_v1.RegionalInput.from_model_fitter(fitter))
```

Comment:
This doesn't look like it's in parallel.  we probably want to run this in a pool right?

Reply:
Done